frontend/packages/console-app/src/components/lightspeed/Lightspeed.scss

```diff
@@ -91,7 +91,6 @@
 
   .pf-v6-theme-dark & {
     background-image: url('logo-dark.svg');
-    background-color: var(--pf-t--global--background--color--primary--default) !important;
```

Comment:
This is in fact needed (note the table border bleeding through).

<img width="77" alt="Screenshot 2025-02-13 at 9 12 07 AM" src="https://github.com/user-attachments/assets/9ff8bab0-e9fd-4321-9d41-7db74276716d" />

Reply:
Ah good catch, when I was testing my page didn't have a border. Added it back in